public_api_registry.json

```diff
@@ -0,0 +1,380 @@
+{
+  "great_expectations.render.components.RenderedContent": [
+    "to_json_dict"
+  ],
```

Comment:
Anything without `__init__` will not be rendered in docs because the parent is not rendered.

We should do one of two things:
1. Add `@public_api` to the parent
2. Remove `@public_api` from the method

Reply:
A test around this that checks these conditions would be easy to write given the registry code above

Reply:
do we have to check this in?

Reply:
nope just showcasing what the current public API looks like